### PR TITLE
add complex example to ParametersIndentationViolation

### LINKED ISSUE #1021: Specific case for ParametersIndentationViolation
I have a lot of violations fro my pytest fixtures like this one
```pytest
@pytest.mark.parametrize(
    ("is_verified", "status_code"),
    [
        (True, web.HTTPOk.status_code),
        (False, web.HTTPUnauthorized.status_code),
    ])
```
It's unclear from doc examples how to fix it. Documentation contains only simple examples.

I suggest adding one complex example like the above one.

### PR DESCRIPTION
**Pull request by AlwxSin**
# I have made things!

## Checklist

- [X] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have updated the documentation for the changes I have made
- [X] I have added my changes to the `CHANGELOG.md`

## Related issues
closes #1021

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ We used to have incremental versioning before `0.1.0`.
 
 - Fixes that `_` was marked as invalid by `VagueImportViolation`
 - Fixes that docs for `VagueImportViolation` were misleading
+- Add more complex example to `ParametersIndentationViolation`
```
Comment (sobolevn) [inline CHANGELOG.md]: Can you please also add #1020 to the list?

File: wemake_python_styleguide/violations/consistency.py [hunk 2]
```diff
@@ -754,6 +754,13 @@ def my_function(
             last_item,
         ], end='')
 
+        # Complex case:
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/consistency.py]: `# Correct complex case:`

File: CHANGELOG.md (hunk 1 above)
Comment (AlwxSin) [reply CHANGELOG.md]: Added. Also added issue numbers, is it ok?

File: wemake_python_styleguide/violations/consistency.py (hunk 2 above)
Comment (AlwxSin) [reply wemake_python_styleguide/violations/consistency.py]: Fixed

**Pull request merged**